composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "cooldogedev/bedrockeconomy",
+    "license": "MIT",
+    "type": "pocketmine-plugin",
+    "scripts": {
+        "analyze": "vendor/bin/phpstan analyze src",
+        "build": "php -dphar.readonly=0 vendor/bin/pharynx -i=. -c -p=BedrockEconomy.phar"
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/cooldogepm/libsql"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/SOF3/Commando"
+        }
+    ],
+    "require": {
+        "pocketmine/pocketmine-mp": "^5.0.0",
+        "cooldogedev/libsql": "dev-pm5",
+	"cortexpe/commando": "dev-patch-1"
```

Comment:
I don't guarantee to keep this branch. Better switch to a better maintained branch in the future.